### PR TITLE
#8246: functional_whisper CG test_demo

### PR DESCRIPTION
**Pull request by kkeerthana0573**
This PR contains test_demo for functional_whisper Conditional Generation.

### CODE REVIEW

**Review by xanderchin — APPROVED** [review]

**Comment by tt-rkim** [comment]
@kkeerthana0573 
Pls review the following errors in nightly pipeline:

```
FAILED tests/ttnn/integration_tests/whisper/test_ttnn_functional_whisper.py::test_encoder[sequence_length=3000-feature_size=80-batch_size=1-model_name=openai/whisper-base-ttnn_model=models.experimental.functional_whisper.tt.ttnn_functional_whisper] - TypeError: encoder() missing 1 required keyword-only argument: 'device'
FAILED tests/ttnn/integration_tests/whisper/test_ttnn_functional_whisper.py::test_ttnn_whisper[ttnn_model=models.experimental.functional_whisper.tt.ttnn_functional_whisper] - TypeError: whisper() missing 1 required keyword-only argument: 'device'
FAILED tests/ttnn/integration_tests/whisper/test_ttnn_optimized_functional_whisper.py::test_encoder[sequence_length=3000-feature_size=80-batch_size=1-model_name=openai/whisper-base-ttnn_model=models.experimental.functional_whisper.tt.ttnn_optimized_functional_whisper] - TypeError: encoder() missing 1 required keyword-only argument: 'device'
FAILED tests/ttnn/integration_tests/whisper/test_ttnn_optimized_functional_whisper.py::test_ttnn_whisper[ttnn_model=models.experimental.functional_whisper.tt.ttnn_optimized_functional_whisper] - TypeError: whisper() missing 1 required keyword-only argument: 'device'
```

Thank you for running. Don't forget to encourage your fellow MCW team members to continue such practices

**Comment by kkeerthana0573** [comment]
@tt-rkim,
Both the tests are updated. 
All whisper integration tests are updated in [#8339](https://github.com/tenstorrent/tt-metal/pull/8339). To avoid further conflicts, I shall merge this PR after #8339 is merged.
Thanks.

**Comment by tt-rkim** [comment]
You never merged this? Do we not want this anymore?

@eyonland

**Pull request closed without merging**
